src/packs.c

```diff
@@ -111,11 +110,15 @@ int download_subscribed_packs(struct list *subs, bool required)
 		iter = iter->next;
 
 		if (sub->oldversion == sub->version) { // pack didn't change in this release
+			fprintf(stderr, "no changes for %s pack for current version (pack %d of %d)\n",
+				sub->component, ++complete, list_length);
 			continue;
 		}
 
+		fprintf(stderr, "%s pack for version %i (pack %d of %d)\n",
```

Comment:
Hmm. The reason why we made the last change to pack progress messages was to avoid printing *anything* when a delta pack does not exist, because this was confusing to users.